addons/lunch/controllers/main.py

```diff
@@ -121,8 +121,8 @@ def _check_user_impersonification(self, user_id=None):
         if (user_id and request.env.uid != user_id and not request.env.user.has_group('lunch.group_lunch_manager')):
             raise AccessError(_('You are trying to impersonate another user, but this can only be done by a lunch manager'))
 
-    def _get_current_lines(self, user_id):
-        return request.env['lunch.order'].search([('user_id', '=', user_id), ('date', '=', fields.Date.today()), ('state', '!=', 'cancelled')])
+    def _get_current_lines(self, user):
```

Comment:
It is forbidden to change the method signature on a stable release. 

But as the 14.3 is not released yet, I supposed this is ok.

Reply:
I understand the stable policy, and we can do it other way by reading again res.users but I would rather make an exception here because we are the only ones concerned and the code is way much better this way.